src/Tizen.Multimedia/AudioManager/AudioDeviceType.cs

```diff
@@ -65,6 +65,11 @@ public enum AudioDeviceType
         /// <summary>
         /// Bluetooth voice (SCO).
         /// </summary>
-        BluetoothVoice
+        BluetoothVoice,
+
+        /// <summary>
+        /// Network.
```

Comment:
Description is too short to understand,
What kind of network protocol was used?

Reply:
This device type will be used when ACM service which is from VD/MCD/public spk profile is activated internally. One another example, PulseAudio - tizen audio backend - supports RAOP protocol and we can use the protocol only with internal test application.(for now, not sure about exporting public API). It would be represented via this new device type.

You can easily think this kind of device is used when audio data is transmitted via wifi network.

Reply:
Would you please update the description to summarize this?

Reply:
It is android reference for you
https://developer.android.com/reference/android/media/AudioDeviceInfo#TYPE_IP